tinyfilemanager.php

```diff
@@ -1582,7 +1586,12 @@ class="edit-file"><i class="fa fa-pencil-square-o"></i> <?php echo lng('Advanced
             } elseif ($is_video) {
                 // Video content
                 echo '<div class="preview-video"><video src="' . fm_enc($file_url) . '" width="640" height="360" controls preload="metadata"></video></div>';
-            } elseif ($is_text) {
+	    } elseif ($is_md) {
+                echo '<pre class="x-mdhere">';
+                echo $content;
+                echo "</pre>";
+                echo "<script>show_md();</script>";
```

Comment:
This will throw a Javascript error that `show_md()` doesn't exists. The reason is that this method hasn't been defined yet. It is defined in the footer